app/controllers/PaymentDueController.scala

```diff
@@ -63,7 +63,7 @@ class PaymentDueController @Inject()(val checkSessionTimeout: SessionTimeoutPred
         paymentDueService.getWhatYouOweChargesList().map {
           whatYouOweChargesList =>
             Ok(views.html.whatYouOwe(chargesList = whatYouOweChargesList, currentTaxYear = user.incomeSources.getCurrentTaxEndYear,
-              paymentEnabled = isEnabled(Payment), implicitDateFormatter = dateFormatter, backUrl = backUrl, user.saUtr)).addingToSession(SessionKeys.chargeSummaryBackPage -> "paymentDue")
```

Comment:
Please remove this from the view instead of setting it to true

Reply:
done